<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сотрудники</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Продление пропусков</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_all">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Всем сотрудникам</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_ok">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Печать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_cancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="d_note">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Служебка от:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="number">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Исх.:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_open">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>440</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>20</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Шаблон</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Новый</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_save">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>440</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_kill">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить шаблон</string>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_6">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_9">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_month">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Месяц</string>
   </property>
  </widget>
  <widget class="QComboBox" name="worker_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
